lms/templates/course_modes/track_selection.html

```diff
@@ -16,6 +16,14 @@
 
 <%block name="js_extra">
     <script type="text/javascript">
+        // upgrade button re-direct to ecommerce basket (which re-directs to Payment MFE)
+        % if use_ecommerce_payment_flow:
+        $('button[name=verified_mode]').click(function(e){
```

Comment:
Does this need to be inside a `$()`? I noticed that this function happens to be inside a [`$(document).ready()`](https://learn.jquery.com/using-jquery-core/document-ready/) in `choose.html` and didn’t know if we’ve tested if we need the same here.

Reply:
It's the JQuery selector for the upgrade button, it's needed. I could add it inside `$(document).ready()` to wait for the DOM to load to be safer, however, if the button is not loaded the user wouldn't be able to click on it?
I've tested locally.

Reply:
Turns out with it it does not work locally. It redirects to ID verification

Reply:
Ok, thanks for checking!

Reply:
Update on this - Phil's local dev works with adding the shorthand version of `$(document).ready()`.